94-appendixD.Rmd

```diff
@@ -815,5 +813,192 @@ library(ggplot2)
+& \qquad b_{\text{Euro}}\cdot\mathbb{1}_{\mbox{Euro}}(x) + b_{\text{Ocean}}\cdot\mathbb{1}_{\mbox{Ocean}}(x)\\
+&= 3089 + 7914\cdot\mathbb{1}_{\mbox{Amer}}(x) + 9384\cdot\mathbb{1}_{\mbox{Asia}}(x) + \\
+& \qquad 21965\cdot\mathbb{1}_{\mbox{Euro}}(x) + 26721\cdot\mathbb{1}_{\mbox{Ocean}}(x)
+\end{aligned}
+$$
+
+<!--
+TODO: Albert will double check Starry's interpretation:
+-->
+In our previous exploratory data analysis, it seemed that continent is a statistically significant predictor for an area's GDP. Here, by fit a new linear regression using `lm(gdpPercap ~ continent, data = gapminder2007)` where `gdpPercap` is the new outcome variable $y$, we are able to write an equation to predict `gdpPercap` using the continent as statistically significant predictors. Therefore, the regression results matches with the results from your previous exploratory data analysis. 
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Using either the sorting functionality of RStudio's spreadsheet viewer or using the data wrangling tools you learned in Chapter \@ref(wrangling), identify the five countries with the five smallest (most negative) residuals? What do these negative residuals say about their life expectancy relative to their continents?
+
+**Solution**: 
+Using the sorting functionality of RStudio's spreadsheet viewer, we can identify that the five countries with the five smallest (most negative) residuals are: Afghanistan, Swaziland, Mozambique, Haiti, and Zambia. 
+
+These negative residuals indicate that these data points have the biggest negative deviations from their group means. This means that these five countries' average life expectancies are the lowest comparing to their respective continents' average life expectancies. For exampple, the residual for Afghanistan is $-26.900$ and it is the smallest residual. This means that the average life expectancy of Afghanistan is $26.900$ years lower than the average life expectancy of its continent, Asia. 
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Repeat this process, but identify the five countries with the five largest (most positive) residuals. What do these positive residuals say about their life expectancy relative to their continents?
+
+**Solution**: 
+Using either the sorting functionality of RStudio's spreadsheet viewer, we can identify that the five countries with the five largest (most positive) residuals are: Reunion, Libya, Tunisia, Mauritius, and Algeria. 
+
+These positive residuals indicate that the data points are above the regression line with the longest distance. This means that these five countries' average life expectancies are the highest comparing to their respective continents' average life expectancies. For exampple, the residual for Reunion is $21.636$ and it is the largest residual. This means that the average life expectancy of Reunion is $21.636$ years lower than the average life expectancy of its continent, Africa. 
```

Comment:
"are above the regression line ~with the longest distance~ by the largest amount"